### PR TITLE
feat: TimePicker Material Style

### PR DESCRIPTION
**Pull request by MaximeDion-Work**
﻿GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Feature

## Description

Adding Material style for TimePicker
image

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] Tested UWP
- [x] Tested iOS
- [x] Tested Android
- [ ] ~~Tested WASM~~
- [ ] ~~Tested MacOS~~
- [x] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Content/Controls/TimePickerSamplePage.xaml [hunk 1]
```diff
@@ -16,9 +16,33 @@
...
+			<smtx:XamlDisplay UniqueKey="TimePickerSamplePage_Default"
+							  Style="{StaticResource XamlDisplayBelowStyle}">
+
+				<TimePicker Header="Select a Time"
+							Style="{StaticResource MaterialTimePickerStyle}"
+							ios:FlyoutPresenterStyle="{StaticResource MaterialTimePickerFlyoutPresenterStyle}" />
+			</smtx:XamlDisplay>
+
+			<!-- Disbaled -->
+			<TextBlock Text="Disbaled"
```
Comment (Xiaoy312) [inline src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Content/Controls/TimePickerSamplePage.xaml:34]: typo x2

File: src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Content/Controls/TimePickerSamplePage.xaml [hunk 2]
```diff
@@ -16,9 +16,33 @@
...
+					   Style="{StaticResource MaterialSubtitle2}"
+					   Margin="12,8,12,4" />
+			
+			<!-- TimePicker -->
+			<smtx:XamlDisplay UniqueKey="TimePickerSamplePage_Disabled"
+							  Style="{StaticResource XamlDisplayBelowStyle}">
+
+				<TimePicker Header="Select a Time"
+							Style="{StaticResource MaterialTimePickerStyle}"
+							ios:FlyoutPresenterStyle="{StaticResource MaterialTimePickerFlyoutPresenterStyle}"
```
Comment (Xiaoy312) [inline src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Content/Controls/TimePickerSamplePage.xaml:44]: this line should be part of MaterialTimePickerStyle setters

**Comment by Xiaoy312** [comment]
image
thats probably unacceptable

**Comment by MaximeDion-Work** [comment]
> image
> thats probably unacceptable
> we should set the visibility via binding=converter or from "HasTimeStates" states

Fixed thought there are disparities between iOS / Android and UWP
image

File: src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Content/Controls/TimePickerSamplePage.xaml [hunk 3]
```diff
@@ -16,9 +16,32 @@
...
```
Comment (XiaotianNetlift) [inline src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Content/Controls/TimePickerSamplePage.xaml:43]: L43 not necessary anymore

**Review by XiaotianNetlift — APPROVED** [review]

**Pull request merged**